spec/system/support_interface/change_job_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.feature 'Change job' do
+  include DfESignInHelpers
+
+  scenario 'Change the english GCSE on an application form', :with_audited do
```

Comment:
```suggestion
  scenario 'Change the work history on an application form', :with_audited do
```

Reply:
Woopps I copied the first 5 lines of your test 😆